### PR TITLE
Error messages from Scheduler Service during dump import #8880

### PR DESCRIPTION
**Pull request by vbradnitski**

### CODE REVIEW

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java [hunk 1]
```diff
@@ -78,7 +81,8 @@
 {
     private static final Logger LOG = LoggerFactory.getLogger( DumpServiceImpl.class );
 
-    private static final RepositoryId AUDIT_LOG_REPO_ID = RepositoryId.from( "system.auditlog" );
+    private static final Set<RepositoryId> SYSTEM_REPO_IDS =
```
Comment (rymsha) [inline modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:84]: it is possible to generify with `RepositoryEntryService.findRepositoryEntryIds` and filter repos that start with `system.`

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java [hunk 2]
```diff
@@ -361,7 +371,10 @@ public SystemLoadResult load( final SystemLoadParams params )
             this.eventPublisher.publish( RepositoryEvents.restored() );
             LOG.info( "Dump Load completed" );
         }
-        catch ( IOException e )
+        catch (
```
Comment (rymsha) [inline modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:374]: something went wrong with formatting here

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java (hunk 1 above)
Comment (vbradnitski) [reply modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:84]: Do we prohibit the creation of custom repos with `system.*` names?

Comment (rymsha) [reply modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:84]: Actually, yes. Or we won't be able to freely use it ourselves.
Anyone who creates system.* steps into muddy waters.

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java [hunk 3]
```diff
@@ -78,7 +81,8 @@
 {
     private static final Logger LOG = LoggerFactory.getLogger( DumpServiceImpl.class );
 
-    private static final RepositoryId AUDIT_LOG_REPO_ID = RepositoryId.from( "system.auditlog" );
+    private static final Set<RepositoryId> SYSTEM_REPO_IDS =
+        Set.of( RepositoryId.from( "system.auditlog" ), RepositoryId.from( "system.scheduler" ) );
```
Comment (rymsha) [inline modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:85]: List (for order predictability) would be slightly better

File: modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java [hunk 4]
```diff
@@ -314,16 +318,18 @@ public SystemLoadResult load( final SystemLoadParams params )
             final RepositorySettings currentSystemSettings =
                 repositoryEntryService.getRepositoryEntry( SystemConstants.SYSTEM_REPO_ID ).getSettings();
 
-            final RepositorySettings currentAuditLogSettings = repositoryEntryService.getRepositoryEntry( AUDIT_LOG_REPO_ID ).getSettings();
+            final Map<RepositoryId, RepositorySettings> repoSettings = new HashMap();
+
+            SYSTEM_REPO_IDS.forEach( repo -> repoSettings.put( repo, repositoryEntryService.getRepositoryEntry( repo ).getSettings() ) );
```
Comment (rymsha) [inline modules/core/core-repo/src/main/java/com/enonic/xp/repo/impl/dump/DumpServiceImpl.java:323]: use Collector instead

**Review by rymsha — APPROVED** [review]

**Review by anatol-sialitski — APPROVED** [review]

**Pull request merged**